### PR TITLE
Update metadata, version, and changelog for 645 area code

### PR DESCRIPTION
**Pull request by ankita-ac**
Updating metadata, version, and changelog to fix invalid area code for 645

### CODE REVIEW

**Review by jaykravetz — APPROVED** [review]

File: .DS_Store
Comment (ankit-tangoe) [inline .DS_Store:1]: Please remove this file

File: data/.DS_Store
Comment (ankit-tangoe) [inline data/.DS_Store:1]: Please remove this file.

**Review by ankit-tangoe — CHANGES_REQUESTED** [review]

File: data/.DS_Store
Comment (ankita-ac) [reply data/.DS_Store:1]: Resolved

File: .DS_Store
Comment (ankita-ac) [reply .DS_Store:1]: Resolved

**Review by ankit-tangoe — APPROVED** [review]
LGTM!

**Pull request merged**